### PR TITLE
Feature/scope change v3

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Api/Search/PCS/Types/query.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface Query {
+    DocumentNo: string;
+    Id: number;
+    DocumentType__Code?: any;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/PCS/Types/query.ts:4]: unknown possibly

File: src/apps/ScopeChangeRequest/Api/Search/PCS/checkMyFunctionalRole.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { BaseClient } from '../../../../../../packages/httpClient/src';
+import { FunctionalRole } from './Types/functionalRole';
+
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/PCS/checkMyFunctionalRole.ts:3]: docs

File: src/apps/ScopeChangeRequest/Api/Search/PCS/searchCommPkg.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { BaseClient } from '@equinor/http-client';
+import { CommPkg } from './Types/commpkg';
+import { TypedSelectOption } from '../searchType';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/PCS/searchCommPkg.ts:3]: convert to hook

File: src/apps/ScopeChangeRequest/Api/Search/PCS/searchPerson.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { BaseClient } from '../../../../../../packages/httpClient/src';
+import { TypedSelectOption } from '../searchType';
+
+interface Person {
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/PCS/searchPerson.ts:4]: export to types

File: src/apps/ScopeChangeRequest/Api/Search/STID/Types/Document.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    docCategory: string;
+    docType: string;
+    contrCode: string;
+    revNo: string;
+    revDate: Date;
+    revStatus: string;
+    revisionProject: string;
+    reasonForIssue: string;
+    remark?: any;
+    tagNoMedia?: any;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/STID/Types/Document.ts:20]: any??

File: src/apps/ScopeChangeRequest/Api/Search/STID/searchStid.ts [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+export type StidTypes = 'document';
+
+/**
+ *
+ * @param searchString
+ * @param searchItem
+ * @param procosysClient
+ * @returns
+ */
+export const searchPcs = async (
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/Search/STID/searchStid.ts:14]: rename

File: src/apps/ScopeChangeRequest/Api/getOrigins.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
...mZqT3BHVVRxdk9uS1F0Q3pidmljNnNGdWF0ekhrenB3U0N4N0FkeXJ2TncvRVJ2dUlLN040dzlPRjRjWHhsTWx1aEdFd0lZPSIsImFtciI6WyJyc2EiLCJtZmEiXSwiYXBwaWQiOiIzYmVjYzY5Zi01MTBhLTQxMWItOTJhNC1jMGJmOGQ1Y2E1ODgiLCJhcHBpZGFjciI6IjAiLCJmYW1pbHlfbmFtZSI6IkVpa2FhcyIsImdpdmVuX25hbWUiOiJHdXN0YXYiLCJpcGFkZHIiOiIxMjguMzkuNjEuMjAzIiwibmFtZSI6Ikd1c3RhdiBFaWthYXMiLCJvaWQiOiJkZmVhYThkZS05MmQyLTRlZTQtYjE3MS02Y2FiYTBjMTYzYmYiLCJvbnByZW1fc2lkIjoiUy0xLTUtMjEtMjIwNTIzMzg4LTEwODUwMzEyMTQtNzI1MzQ1NTQzLTI2MTY4MTMiLCJyaCI6IjAuQVFJQU5hS2tPdUsyMVVpUmxYX1BCYlJac0pfRzdEc0tVUnRCa3FUQXY0MWNwWWdDQU5VLiIsInNjcCI6IkZpbGVzLnJlYWR3cml0ZSIsInN1YiI6ImlPXzM4T2pBX1BhZ0I3MmlWYjhFTDd2QTBvcWVEZWlOYm5vR3NzS2ZSemsiLCJ0aWQiOiIzYWE0YTIzNS1iNmUyLTQ4ZDUtOTE5NS03ZmNmMDViNDU5YjAiLCJ1bmlxdWVfbmFtZSI6IkdVRUlAZXF1aW5vci5jb20iLCJ1cG4iOiJHVUVJQGVxdWlub3IuY29tIiwidXRpIjoiZlVXM25YWjZEVU9PS0JackpkY0dBZyIsInZlciI6IjEuMCJ9.Vqn5Y_iq-0uFSXfu1ZJO0pqHY9024b5mtj4BoCoXWxmz2Of9syMDDF0DWLjz4zxBXyMv80WnzHcnNEIKJ93_cNZS-pWyIseiadjzRDeREI1c4JtQc-dlgonAYUBQwrACYP5KoW5ppIxL6Tg7CHT1Mh1h5DCfLe_70nQmXbKX5RU6ekMjBdczoDSM_FnUjkY_Js6i6nzfSXQ_6dnNjSc1F5tD81fDNkp29xQ_v0HkLZ9tYRdUohCJyjgc7SAve6-dieEO-2SESug74WTvK6wCG96Bej6G3hyjjl4RGfcxCGuW44I4N2rYK9p8U7nfmhn5qjXS7DKJDuV1v9RHkaWRdQ',
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/getOrigins.ts:16]: remove

File: src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx [hunk 8]
```diff
@@ -1,63 +1,88 @@
 import { Button, Icon } from '@equinor/eds-core-react';
 import { GeneratedForm, useForm } from '@equinor/Form';
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
 import { scopeChangeRequestSchema } from '../../Schemas/scopeChangeRequestSchema';
 import { ScopeChangeRequest } from '../../Types/scopeChangeRequest';
 import { useMutation } from 'react-query';
 import { getScopeChangeById, postScopeChange } from '../../Api/';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx:8]: Barrel export

File: src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx [hunk 9]
```diff
@@ -1,63 +1,88 @@
...
 
 export const ScopeChangeRequestForm = ({
     closeScrim,
     setHasUnsavedChanges,
 }: ScopeChangeRequestFormProps): JSX.Element => {
-    const formData = useForm<ScopeChangeRequest>(scopeChangeRequestSchema);
-    const [scID, setScID] = useState<string | undefined>(undefined);
+    const formData = useForm<ScopeChangeRequest>(scopeChangeRequestSchema, {
+        category: 'Hidden carryover',
+        origin: 'NCR',
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx:35]: remove

File: src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx [hunk 10]
```diff
@@ -103,3 +157,7 @@ const TitleHeader = styled.div`
 `;
 
 const FormContainer = styled.div``;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx:159]: delete

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/applyEds.tsx [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+        //Text color, make grey
+        neutral50: `${tokens.colors.text.static_icons__secondary.rgba}`,
+        neutral60: `${tokens.colors.interactive.primary__resting.rgba}`,
+        neutral70: 'blue',
+        neutral80: `${tokens.colors.interactive.primary__resting.rgba}`,
+    },
+});
+
+export const applyEdsStyles = () => {
+    return {
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/applyEds.tsx:44]: remove

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/cancellationToken.ts [hunk 12]
```diff
@@ -0,0 +1,35 @@
+const CANCEL = Symbol();
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/cancellationToken.ts:1]: delete file

File: src/apps/ScopeChangeRequest/Schemas/scopeChangeRequestSchema.ts [hunk 13]
```diff
@@ -41,29 +33,38 @@ export const scopeChangeRequestSchema: Schema<ScopeChangeRequest> = {
...
 };
 
 const getCategories = async (): Promise<string[]> => {
     let selectOptions: string[] = [];
     const requestOptions = {
         method: 'GET',
-        headers: { 'Content-Type': 'application/json' },
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization:
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Schemas/scopeChangeRequestSchema.ts:65]: remove

File: src/packages/Form/src/Components/Form.tsx [hunk 14]
```diff
@@ -45,7 +51,14 @@ export const GeneratedForm = <T, K extends keyof T>({
             {groupedFields.map((fieldArray, index) => {
                 return (
                     <SectionRow key={fieldArray.toString() + index.toString()}>
-                        {fieldArray.map((field: Field<T[K]>, index: number) => {
+                        {fieldArray.map((field: Field<T[K] | CustomField>, index: number) => {
```
Comment (Gustav-Eikaas) [inline src/packages/Form/src/Components/Form.tsx:54]: use IS function

File: src/packages/Form/src/Types/field.ts [hunk 15]
```diff
@@ -16,4 +16,12 @@ export interface Field<Type> {
     order: number;
     inputType: InputType;
     setValue: Setter<Type>;
+    placeholderText: string;
+}
+
+export interface CustomField {
+    title: string;
```
Comment (Gustav-Eikaas) [inline src/packages/Form/src/Types/field.ts:23]: Type up component

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**